game-core/src/main/java/games/strategy/engine/framework/startup/ui/editors/SelectAndViewEditor.java

```diff
@@ -61,10 +61,8 @@ public SelectAndViewEditor(final String labelTitle, final String defaultHelp) {
         new Insets(0, 0, 0, 0), 0, 0));
     selector.setRenderer(new DisplayNameComboBoxRender());
     selector.addItemListener(e -> {
-      if (e.getStateChange() == ItemEvent.SELECTED) {
```

Comment:
Does removing this check cause a double update?  That is, does an `UNSELECTED` event get raised in conjunction with the `SELECTED` event?

I'm assuming you had to do this because, now that you're now using `null` as a list item rather than a _Null Object_, a `SELECTED` event is not fired when "disabled" (`null`) is selected?

Reply:
That's exactly the case.
For the non-null items it's SELECTED and for the null item it's UNSELECTED.

Reply:
And to answer your question: No, events are not fired twice, I was debugging this code earlier and I don't think I remember having hit a breakpoint 2 times for one click, but I can check again to be completely sure.

Reply:
I found a case where the listener is fired twice, when the forum poster is swapped to something else, the old one gets deselected, and the new one gets selected.
I fixed this by checking if the new item is null and restoring the old behaviour as well.